tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_height_weight_test.py

```diff
@@ -8,13 +66,259 @@
 # Third party imports
 from google.api_core.exceptions import ClientError
 
-# Project imports
-import bq_utils
 import constants.cdr_cleaner.clean_cdr as cdr_consts
 from app_identity import PROJECT_ID
 from cdr_cleaner.clean_cdr_engine import generate_job_config
-from cdr_cleaner.cleaning_rules.clean_height_weight import CleanHeightAndWeight
-from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+from cdr_cleaner.cleaning_rules.clean_height_weight import (
+    CleanHeightAndWeight,)
+from common import JINJA_ENV
+# Project Imports
+from common import MEASUREMENT
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import \
+    BaseTest
+
+# The existing person table is created and partitioned on the pseudo column _PARTITIONTIME,
+# partitioning by _PARTITIONTIME doesn't work using a query_statement for creating a table,
+# therefore CREATE OR REPLACE TABLE doesn' work and we need to DROP the table first.
+MEASUREMENT_DATA_TEMPLATE = JINJA_ENV.from_string("""
+DROP TABLE IF EXISTS
+  `{{project_id}}.{{dataset_id}}.measurement`;
+CREATE TABLE
+  `{{project_id}}.{{dataset_id}}.measurement` AS (
+  WITH
+    w AS (
+    SELECT
+      ARRAY<STRUCT<measurement_id INT64,
+      person_id INT64,
+      measurement_concept_id INT64,
+      measurement_date DATE,
+      measurement_datetime TIMESTAMP,
+      measurement_type_concept_id INT64,
+      operator_concept_id INT64,
+      value_as_number FLOAT64,
+      value_as_concept_id INT64,
+      unit_concept_id INT64,
+      range_low FLOAT64,
+      range_high FLOAT64,
+      provider_id INT64,
+      visit_occurrence_id INT64,
+      measurement_source_value STRING,
+      measurement_source_concept_id INT64,
+      unit_source_value STRING,
+      value_source_value STRING>>
+      [(1,1,3036277,"2017-03-25","2017-03-25 01:00:00 UTC",44818701,4172703,5.6,0,9330,null,null,null,1,"8302-2",3036277,"in",""),
+        (2,2,3036277,"2017-10-18","2017-10-18 01:00:00 UTC",44818701,4172703,62.5,0,9330,null,null,null,2,"8302-2",3036277,"in",""),
+        (3,3,3036277,"2017-02-07","2017-02-07 01:00:00 UTC",44818701,4172703,1.8,null,8582,null,null,null,3,"8302-2",3036277,"cm",""),
+        (4,4,3036277,"2017-01-06","2017-01-06 01:00:00 UTC",44818701,4172703,193.0,0,8582,null,null,null,4,"8302-2",3036277,"cm",""),
```

Comment:
Is there a reason this cannot use `INSERT INTO ... VALUES ...`?

Reply:
The INSERT INTO in the bigquery base expects all the required columns to be added to the data. and the person table expects partiotined. So used this method to load only the columns that were needed.

Reply:
Aren't all the required columns being added here already?

Reply:
only in the measurement table,  for the other tables no. So, to maintain consistency with all the inserting queries I choose to use a similar format.

Reply:
I see, that clears it up